modules/bot.py

```diff
@@ -0,0 +1,192 @@
+        super().__init__(logger, commandhandlers)
+
+        self.__trello_client = TrelloClient(
+            api_key=TRELLO_API_KEY,
+            api_secret=TRELLO_API_SECRET,
+        )
+
+    def _access_feedback_list(self):
+        """
+        If properly configured, get the Trello feedbacks list.
+        :return: trello.trellolist.List
+        """
+        try:
+            for board in self.__trello_client.list_boards():
+                if board.name == TRELLO_FEEDBACK_BOARD:
+                    break
+
+            if not board:
+                print("Trello is not configured properly: board not found.")
+                return None
+
+            for liste in board.list_lists():
+                if liste.name == TRELLO_FEEDBACK_LIST:
+                    break
+
+            if not liste:
+                print("Trello is not configured properly: list not found.")
+                return None
+        except:
+            print("Trello is not configured properly: invalid credentials.")
+            return None
+
+        return liste
+
+    def _add_feedback_to_trello(self, feedback):
+        """
+        Add a new feedback to the Trello feedbacks list.
+        :param feedback: (title: String, description: String)
+        :return: True if added successfully, else False.
+        """
+        liste = self._access_feedback_list()
+
+        if liste:
+            content, description = feedback
+            liste.add_card(content, desc=description)
+            return True
+        else:
+            return False
+
+    def _get_feedbacks_from_trello(self):
+        """
+        Get the first `MAX_FEEDBACKS` the feedbacks from the Trello feedbacks list (titles only).
+        :return: String if Trello active, else None.
+        """
+        liste = self._access_feedback_list()
+
+        if liste:
+            retour = []
+            remaining = 0
+
+            cards = liste.list_cards_iter()
+            for i, card in enumerate(cards):
+                retour.append("{}. {}".format(i + 1, card.name))
+                if i == self.MAX_FEEDBACKS - 1:
+                    break
+            for _ in cards:
+                remaining += 1
+            if remaining > 0:
+                retour.append("... And {} more!".format(remaining))
+
+            return retour
+        else:
+            return None
+
+    def start(self, update: Update, context: CallbackContext) -> None:
+        """
+        Says hello to the user and auto-enable the reply mode.
+        """
+        user = update.effective_user
+        update.message.reply_markdown_v2(
+            fr"Bonjour {user.mention_markdown_v2()} \!",
+            reply_markup=ForceReply(selective=True),
+        )
+
+        self.logger.info("{} says hi!".format(user.first_name))
+
+    def help_command(self, update: Update, context: CallbackContext) -> None:
+        """
+        Don't forget to update this manually.
+        """
+
+        text = """Available commands:

```

Comment:
doesn't seem to scale well, nor it is mistake-proof. No better way?

Reply:
Yeah for the "help" I wanted to do it in a second time

Because of course it scales ZERO (^:

Reply:
See #10